.github/workflows/integ_test.yml

```diff
@@ -27,6 +27,14 @@ jobs:
           aws_role_to_assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
           aws_region: ${{ secrets.AWS_REGION }}
           aws_s3_bucket: ${{ secrets.AWS_S3_BUCKET_INTEG }}
+      
+      - name: Verify copy resources for v2
```

Comment:
Do we still need the previous "Verify copy resources" step on this branch?

Won't the new V2 step just override everything anyway?

Reply:
we still need those for datastore and api, so keeping it for now. Will remove them when all the plugins are moved to v2 sturcture.

Reply:
But datastore and api are not run on this workflow 🤔 

Just to clarify, I'm taking about removing the _step_ from this job, not removing the actual resources from AWS

Reply:
ah got it! thanks for clarifying, removed the old step